starter.js

```diff
@@ -0,0 +1,203 @@
+
+var name = 'Alanoud';
+function justDoit(name ){
+      console.log(name+ ' JUST DO IT!');
+   }
+
+
+//----------------part1--------------------//
+
+function bigOrSmallString(words)
+{
+   if (words.length   > 10)
+   {
+     console.log("This word is loooooong!") ;
+}
+else {
+  console.log( " The word is short ");
+}
+}
+  bigOrSmallString(" AlanouBinSalamah");
+  bigOrSmallString(" Alanoud");
+
+
+//----------------Part2--------------------//
+
+
+function oddOrEvenString (str){
+    
+    if(str.length % 2 ){
+          console.log("This string's length is odd!") ;
+    }else {
+          console.log("This string length is TOTALLY even!") ;
+    }
+}
+
+oddOrEvenString ("Nouf");
+oddOrEvenString ("Abeer");
+
+
+//----------------Part3--------------------//
+
+                                               
+  
+var numbers = [ 34 , 16 , 10 , 70 , 45];
+
+  function medianOfArray(median){
+     median.so( function ( num1 , num2){ // I have a PROBLEM here
+         return num1 - num2;
+       });
+      }
+
+     var med = Math.floor(numbers.length/2);
+
+      if(median.length % 2){
+        return median[med];
+
+    } else{
+          return (median[med-1] + median[med]) / 2;
+      }
+     
+      medianOfArray(numbers);
+
+//----------------Part4--------------------//
+
+                                               
+var sum = [ 3 ,5 ,1 , 6, 9].reduce(add , 0);
+
+function sumArray( num1 , num2){
+    
+     return num1 + num2;
+    }
+
+//console.log(sum);
+
+//----------------Part5--------------------//
+
+                                   
+function vowelCount(vowel){
+
+   var count = 0;
+
+for(var i=0 ; i < vowel.length ; i++){
+
+    if(vowel[i] == "a"||vowel[i] == "e"|| vowel[i] == "i"||vowel[i] == "u"||vowel[i] == "o"){
+
+     return count += 1;
+}
+
+}
+  return count;  
+}
+console.log(vowelCount ( 'lead'));
+
+//----------------Part6--------------------//
+                                      
+
+function initials(init){
+
+    var name = init.split('');
+    var initName  = '';
+for( var i=0; i<name.length; i++){
+    if(name[i].length>0 && name[i]!=''){
+
+      return  initName += name[i][0];
+    }else{
+      return initName ;
+    }   
+}
+      
+}
+console.log(initials("Alanoud Mohammed "));
+
+//----------------Part7--------------------//
+
+
+
+
+
+                                          
+//var day= prompt("Please enter the day","")// not defined also window.alert is not defined
+function week (day){                        
+if(day=="Monday")
+    console.log("Energize!");
+
+else if(day=="Tuesday")
+    console.log("Just getting started!");
+
+ else if(day=="Wednesday")
+    console.log("Hump Hump!");
+
+ else if(day=="Wednesday")
+    console.log("Hump Hump!");
+
+else if(day=="Thursday")
+    console.log("Almost there!");
+
+else if(day=="Friday" || day=="Saturday" || day=="Sunday")
+    console.log("Weeeeeee!");
+
+}
+week("Sunday")
+//----------------Part8--------------------//
```

Comment:
you have some logic problems in this part please remind me if you want me to explain it further